frontend/app/forms/motion-to-reduce-conviction/motion-to-reduce-conviction.pdf.component.js

```diff
@@ -1,5 +1,136 @@
+            {renderData("person.lastName")}
+          </div>
+        </PositionedString>
+        <PositionedString
+          dataKey="person.addressStreet"
+          left="11.65%"
+          top="17.49%"
+        />
+        <PositionedString
+          debugKey="petitionerAddress"
+          left="11.65%"
+          top="21.09%"
+        >
+          {data.person.addressCity && `${renderData("person.addressCity")}, `}
+          {renderData("person.addressState")}
+          {`\u0020`}
+          {renderData("person.addressZip")}
+        </PositionedString>
+        <PositionedString
+          dataKey="person.homePhone"
+          left="11.65%"
+          top="24.5%"
+        />
+        <PositionedString dataKey="person.email" left="11.65%" top="28.1%" />
+        <PositionedCheckmark
+          debugKey="petitionerIsRep"
+          left="20%"
+          top="31.5%"
+          shouldShow={data.person.petitionerRepresentative === "petitioner"}
+        />
+        <PositionedCheckmark
+          debugKey="attorneyIsRep"
+          left="20%"
+          top="33%"
+          shouldShow={data.person.petitionerRepresentative === "attorney"}
+        />
+        <PositionedString
+          dataKey="person.petitionerBarNumber"
+          left="62%"
+          top="33%"
+        />
+        <PositionedCheckmark
+          debugKey="courtTypeDistric"
+          left="36%"
+          top="36.9%"
+          shouldShow={data.case.courtType === "District"}
+        />
+        <PositionedCheckmark
+          debugKey="courtTypeJustice"
+          left="46.9%"
+          top="36.9%"
+          shouldShow={data.case.courtType === "Justice"}
+        />
+        <PositionedString dataKey="case.courtAddress" left="27.8%" top="43%" />
+        <PositionedString left="25.9%" top="40.2%" debugKey="judicialDistrict">
+          {data.case.courtAddress &&
+            getJudicialDistrict(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString left="50.7%" top="40.2%" debugKey="county">
+          {data.case.courtAddress &&
+            getCounty(data.case.courtAddress, data.case.courtType)}
+        </PositionedString>
+        <PositionedString dataKey="case.caseNumber" left="54.1%" top="57.9%" />
+        <PositionedString dataKey="case.judgeName" left="54.1%" top="62.5%" />
+        <PositionedString dataKey="case.plaintiff" left="11%" top="50.7%" />
+        <PositionedString dataKey="case.defendant" left="11%" top="58.1%" />
+        {requestedConvictionDegreeInformation(
+          data.case.convictionDegree,
+          data.case.reducedConvictionDegree
+        )}
+      </RenderPage>
+    </>
+  );
+}
+
+function requestedConvictionDegreeInformation(degA, degB) {
+  const degree = degreeStringToInt(degA);
+  const requestedDegree = degreeStringToInt(degB);
+  const isReduceByOneDeg = requestedDegree - degree > 1 ? false : true;
```

Comment:
We could just use a conditional operator instead of ternary to get this:

```suggestion
  const isReduceByOneDeg = requestedDegree === 1
```